### PR TITLE
Fix  exposed game credentials

### PR DESCRIPTION
**Pull request by tobybrave**
#### What does this PR do?
remove exposed chess plugin credentials

#### How should this be manually tested?
make changes to `server/src/config/config.json`
`git commit -am"commit content"`
`git push origin _name_`

if changes still persist
`git rm -r --cached server/src/config/config.json`
commit and push again

#### Any background context you want to provide?
credentials used in communicating with the zuri core api was exposed in the project repo

#### Better option
put the credentials in .env

### CODE REVIEW

**Comment by tobybrave** [comment]
i'm working on the backend, why is my commit failing for frontend?

**Comment by tobybrave** [comment]
thank you for putting my react skill to test :).
would create a proper dev ci/cd later

File: server/package.json [hunk 1]
```diff
@@ -22,11 +22,14 @@
     "ws": "^8.2.1"
   },
   "devDependencies": {
-    "nodemon": "^2.0.12"
+    "jest": "^27.1.0",
+    "nodemon": "^2.0.12",
+    "supertest": "^6.1.6"
```
Comment (eni4sure) [inline server/package.json:27]: we don't have tests

File: server/.gitignore [hunk 2]
```diff
@@ -1,3 +1,6 @@
+# credentials to connect with zuri core
+src/config/config.json
```
Comment (eni4sure) [inline server/.gitignore:2]: not needed, the api keys we are using is general

File: .github/workflows/fe_error_check.yml [hunk 3]
```diff
@@ -3,9 +3,11 @@ name: EsLint - Checking For Errors [Frontend]
 # Triggers the workflow on pull requests
 on:
   pull_request:
-  
+    branch: [dev]
+    types: [opened, sychronize, reopened, edited]
+
 jobs:
-  build:
+  linter_job:
```
Comment (eni4sure) [inline .github/workflows/fe_error_check.yml:10]: no one edits the yml files except @ibitolamayowa or @eni4sure

File: server/package.json [hunk 4]
```diff
@@ -22,11 +22,14 @@
...
   },
   "scripts": {
     "start": "node index.js",
     "dev": "nodemon index.js",
+    "server-test": "jest --runInBand",
```
Comment (eni4sure) [inline server/package.json:32]: we don't have tests setup???

**Pull request closed without merging**